models/seed-models.js

```diff
@@ -153,3 +153,26 @@ exports.checkTopicExists = (topic) => {
     else return db.query("SELECT * FROM topics WHERE slug =$1;", [topic])
     
   };
+
+
+  exports.removeComment = (comment_id) => {
+    return db
+    .query(
+        `DELETE FROM comments
+        WHERE comment_id = $1
+        RETURNING *;
+    `, [comment_id])
+  }
+
+  exports.checkCommentExists = (comment_id) => {
+    return db.query("SELECT * FROM comments WHERE comment_id = $1;" , [comment_id])
+    .then((result) => {
+        console.log(result.rows)
+        if(result.rows.length === 0) {
+            return Promise.reject({
+                status: 404,
+                msg: "Unable to find resource"
+            })
+        }
+    })
```

Comment:
Don't really want to say delete this as it looks great - but from the comment above I'd say it doesn't need to be used at this stage